MAP/src/components/StatsBox/StatsBox.js

```diff
@@ -1,19 +1,20 @@
 import React, { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { fetchTotals } from '../../services';
-import './Totals.css';
+import './StatsBox.css';
 import { toBool } from '../../utils/utils';
 
-const separateNumber = (number) => {
-  const nfObject = new Intl.NumberFormat('es-ES');
+const separateNumber = (number, format) => {
+  const nfObject = new Intl.NumberFormat(format);
```

Comment:
I like this change, should definitely keep it. However when I switch languages, the `,` doesn't change to `.` as I would expect _(maybe because the language change doesn't persist?)_

Reply:
It's not working as intended right now since I'm passing `currentLanguage` from `App.js` to it—which is actually a function, not a string—so it's defaulting to `,`. I'm currently doing a bit more reading to understand how to use `react-i18next` properly.